source/blog/2017-12-04-lvm-configuration-the-easy-way.html.md

```diff
@@ -0,0 +1,107 @@
+---
+title: LVM Configuration the Easy Way
+author: jmarks, nsofer
+tags: LVM, VDSM
+date: 2017-12-04 09:00:00 CET
+---
+
+Now oVirt features a simple way to prevent a rebooted host from scanning and then activating logical volumes that are not required directly by the host. Why? Because scanning and activating other logical volumes may cause data corruption, slow boot, and other issues.
+
+The solution is a command that generates a filter, which when ran, will reveal to the host only those logical volumes required directly by it.
+
+READMORE
+
+The new command, `vdsm-tool config-lvm-filter` analyzes the current LVM configuration to decide
+whether a filter should be configured. Then, whenever possible, a filter is automatically generated.
+
+### Scenario 1: An Unconfigured Host
```

Comment:
This must be section level 2 - the title of the page is section level 1 (# heading...), this next tile is (## heading 2). If this looks too big, it should be fixed in the site CSS, not by creating invalid document structure.

Same for all other headings.